docs/config.qmd

```diff
@@ -578,6 +578,14 @@ resume_from_checkpoint:
 # Be careful with this being turned on between different models.
 auto_resume_from_checkpoints: false
 
+## Multimodal section
+# int | tuple[int, int] | None . Size to resize images to, width x height.
```

Comment:
I like the typing indication here in the config docs. I wonder if there's a better way to do this, e.g.,

```
image_size: int | tuple[int, int] | None = None
```

This would just be duplicating the typing indication in the Axolotl config pydantic model(s).

Maybe even better would be to just generate these docs from the Pydantic models themselves. We could use `Field`s with `json_schema_extra` populated with the comments that we have here.

Just an idea. It might help reduce duplication of efforts.

Reply:
Yep, I agree, that's why I'm really excited with the quartodocs :)

Reply:
I'm not sure quartodoc can do it, but it should be relatively easy to do with the script if not!